input/resources/ValueSet-name-context.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ValueSet xmlns="http://hl7.org/fhir">
+  <id value="name-context"/>
+  <meta>
+    <profile value="http://hl7.org/fhir/StructureDefinition/shareablevalueset"/>
+  </meta>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm">
+    <valueInteger value="0"/>
+  </extension>
+  <url value="http://terminology.hl7.org.au/ValueSet/name-context"/>
+  <identifier>
+    <system value="urn:ietf:rfc:3986"/>
+    <value value="urn:oid:2.16.840.1.113883.2.3.4.2.4.34"/>
+  </identifier>
+  <name value="NameContext"/>
+  <title value="Name Context"/>
+  <status value="draft"/>
+  <experimental value="false"/>
+  <description value="The Name Context value set is used to classify suitable uses or purposes of names."/>
```

Comment:
Consider changing 'is used' to 'may be used' to ensure consistency with other value set descriptions.

Reply:
Confirmed fix.